### PR TITLE
Resources, Press pages

### LINKED ISSUE #105: Resources landing page
- [ ] Blue box + heading + intro content
- [ ] Pills
- [ ] Styles for list of Resources

### LINKED ISSUE #107: Press landing page
- [ ] Blue box + heading + intro content
- [ ] Pills
- [ ] Styles for list of Press Releases

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #107 
closes #105 

## What this PR does and doesn't do
- This PR does not have the Filter bubbles, but it has everything else.
- Rhombus! Rhombus is now only on window size over 992px.
- Creates 2 includes for the style and logic of showing items, showing `Show more` after 6 items
- Adds a few CSS utility classes, variables

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@segacy1 Does this `Show more` implementation look good? https://deploy-preview-151--cal-itp-website.netlify.app/resources I don't want the `Show more` to hide after it's open, cos then users won't be able to close it.

File: src/_includes/articles.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% for item in items limit: 6 %}
+  {% include article.html %}
+{% endfor %}
+{% if group.size > 6 %}
+  <details>
+    <summary class="d-block mb-3 pb-4">Show more</summary>
+    {% for item in items offset: 6 %}
+      {% include article.html %}
```
Comment (thekaveman) [inline src/_includes/articles.html:8]: Same comment about explicitly passing `item` to the include.

File: src/resources.html [hunk 2]
```diff
@@ -3,31 +3,40 @@
...
-      </article>
-    {% endfor %}
-    {% unless forloop.last %}
-      <hr/>
-    {% endunless %}
-  {% endfor %}
+      {% for group in groups %}
+        <h2 class="mb-4 mt-5">{{ group.name }}</h2>
+        {% assign items = group.items %}
+        {% include articles.html %}
```
Comment (thekaveman) [inline src/resources.html:33]: Same comment about explicitly passing to include.

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Comment by segacy1** [comment]
@machikoyasuda what do you think about [this design](https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?type=design&node-id=6333-35170&t=djHz5aPCnjITVUUM-4) (show more/show less)? From what I understand, we don't have much content, so I don't think it would get unwieldy.

File: src/_includes/articles.html (hunk 1 above)
Comment (thekaveman) [reply src/_includes/articles.html:8]: @machikoyasuda I don't think this one was updated yet.

File: src/resources.html (hunk 2 above)
Comment (thekaveman) [reply src/resources.html]: Yep this looks good!

**Review by thekaveman — APPROVED** [review]
Code and staging site look good to me.

I feel like we shouldn't block this on the "Show more" UX. For this sprint/iteration we probably don't need it as the content isn't overwhelming for now (and sounds like we are even going to drop a Resource based on a comment from @mrose914).

Thoughts about delaying this UX for the next round?

**Comment by machikoyasuda** [comment]
@thekaveman Sounds good to me.

**Comment by thekaveman** [comment]
Errr... I meant we should remove it entirely. Not a big deal.

**Comment by machikoyasuda** [comment]
@thekaveman Oops. Okay I can do that in a next PR!

**Pull request merged**